front/app/containers/Admin/pages-menu/NavbarItemRow/messages.ts

```diff
@@ -0,0 +1,28 @@
+import { defineMessages } from 'react-intl';
```

Comment:
Very good question. I don't know the answer. I didn't check any internals. Just tested with 1 string and it worked fine. You can also test if you don't trust me. 😀

Reply:
Hah, it's not a matter of trust, I can see the results of the test with my own eyes :slightly_smiling_face: But the test doesn't show that Crowdin somehow understands id renamings.

Reply:
It did, the id was adjusted in Crowdin too and translations where there for every language. Also not sure how it works.

Reply:
Where do you see the updated ids and translations on Crowdin? I don't see them here https://github.com/CitizenLabDotCo/citizenlab/pull/2347/files
(sorry for silly questions, I really have to dedicate some time and learn Crowdin)

Reply:
[Here](https://crowdin.com/translate/citizenlabdotcocitizenlab/5500/en-nlbe?filter=basic&value=0#q=test.) for example. (A bit tricky in a case where we have no new strings, got there via the Content tab and then for my branch click on Progress in the little popup menu):
<img width="1501" alt="Screenshot 2022-07-12 at 08 46 19" src="https://user-images.githubusercontent.com/16427929/178426153-3306d4ad-3f45-4778-958d-1a2b108264a8.png">

So Crowdin somehow updated all strings internally and it sees the same strings in our files, so there's nothing to update and it will not be in the diff of the PR you link to.